src/components/PersonLink.tsx

```diff
@@ -0,0 +1,73 @@
+import classNames from 'classnames';
+import { Person } from '../types';
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import { useContext } from 'react';
+import { PeopleContext } from './PeopleContext';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, sex, born, died, fatherName, motherName } = person;
```

Comment:
To avoid this situation you can name it however you want 
![image](https://github.com/mate-academy/react_people-table-advanced/assets/97075591/b9930ab4-c8b4-43e5-9213-5c3bc09678ce)


and you can use an alias when using a destruction assignment

```suggestion
  const { slug as personSlug, name, sex, born, died, fatherName, motherName } = person;
```